src/labels/CustomLabels.labels

```diff
@@ -2661,6 +2677,14 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>alloObjectUnsupported</shortDescription>
         <value>The Manage Allocations button is only supported from a Campaign, Opportunity, Payment or Recurring Donation record.</value>
     </labels>
+    <labels>
+        <fullName>alloOverallocatedError</fullName>
+        <categories>Allocation, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Error when Allocations are cleared because amounts can't convert to percentages</shortDescription>
+        <value>The amount allocated exceeds the {obj_type} amount. Update Allocation amounts so that the total equals the {obj_type} amount.</value>
```

Comment:
Is there a reason these use `{obj_type}` rather than the typical pattern of `{0}` used with the `format()` method?